lib/foreman_maintain/utils/service/systemd.rb

```diff
@@ -1,17 +1,20 @@
 module ForemanMaintain::Utils
   module Service
     class Systemd < Abstract
-      def initialize(name, priority, _options = {})
+      def initialize(name, priority, options = {})
         super
         @sys = SystemHelpers.new
       end
 
       def command(action, options = {})
         do_wait = options.fetch(:wait, true) # wait for service to start
+        all = @options.fetch(:all, false)
         if do_wait && File.exist?('/usr/sbin/service-wait')
           "service-wait #{@name} #{action}"
         else
-          "systemctl #{action} #{@name}"
+          cmd = "systemctl #{action} #{@name}"
+          cmd += ' --all' if all
+          cmd
```

Comment:
hrmm, i just realized this means that --all will be passed to enable/disable which i'm not sure makes sense

in fact i realized that foreman-maintain even provides enable/disable support....  
which i don't think will do the right thing:

```
systemctl enable 'pulpcore-worker@*'  --all

Created symlink from /etc/systemd/system/multi-user.target.wants/pulpcore-worker@\x2a.service to /usr/lib/systemd/system/pulpcore-worker@.service.
```

so i think it might make more sense if the start/stop commands pass :all in through the options param to command.  And then we need to figure out what to do with enable/disable.  I'm not sure how its supposed to know which pulpworkers to enable if they've been disabled.

Reply:
the simplest thing is to not allow disabling/enabling of pulpcore-workers, but that seems kinda like a cop out.  I'm not entirely sure why this functionality is provided.